app/views/govuk_publishing_components/components/_checkboxes.html.erb

```diff
@@ -19,7 +19,7 @@
         <% end %>
 
         <% if cb_helper.error %>
-          <%= tag.span error, id: "#{id}-error", class: "govuk-error-message" %>
+          <%= tag.p error, id: "#{id}-error", class: "govuk-error-message" %>
```

Comment:
The link in the commit message is slightly wrong, should be `https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md#update-the-html-for-error-messages`